Cargo.toml

```diff
@@ -1,3 +1,6 @@
+[workspace]
+exclude = ["tests/test-kernels", "rusty-*", "libhermit-rs"]
```

Comment:
This `exclude` key should not be necessary, since only path dependencies are default members.

Reply:
Nope, at least `test-kernels` is necessary, as `tests` is a default member, but the kernels need to be compiled completely separately:

```
running 1 test
error: current package believes it's in a workspace when it's not:
current:   /home/bla/Hermit/uhyve/tests/test-kernels/Cargo.toml
workspace: /home/bla/uhyve/Cargo.toml

this may be fixable by ensuring that this crate is depended on by the workspace root: /home/bla/Hermit/uhyve/Cargo.toml
Alternatively, to keep it out of the workspace, add the package to the `workspace.exclude` array, or add an empty `[workspace]` table to the package's manifest.
test new_file_test ... FAILED
```
To be on the safe side, I'd also keep the others.